cluster-components/cloud-platform-live-0-kuberos/kuberos.yml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kuberos-oidc-env
+data:
+  OIDC_ISSUER_URL: "https://moj-cloud-platforms-dev.eu.auth0.com/"
+  OIDC_CLIENT_ID: "uVp00zncGYUyqr5IMkLH3zNpvLbwzzyX"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: templates
+data:
+  kubecfg: |
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - cluster:
+        certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMwekNDQWJ1Z0F3SUJBZ0lNRlRRT01mM1g2ZGRCQjV1TE1BMEdDU3FHU0liM0RRRUJDd1VBTUJVeEV6QVIKQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13SGhjTk1UZ3dOVE13TVRRd016QXhXaGNOTWpnd05USTVNVFF3TXpBeApXakFWTVJNd0VRWURWUVFERXdwcmRXSmxjbTVsZEdWek1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBCk1JSUJDZ0tDQVFFQTZoVnJCQ21qUTVkTHNZdFlTZFJ5dmxFRGZudjdHQitaUWN6bVU4dmNUcmdNcTFBb3NMK0UKVEN6Z296dkNLVEZORUlYamdPSlpOZEFHRXdId3ZnZmFqaXBXQm5DSGFTK1haRkFpRitXbzVOYk91MDRYb2p1QwoyT3h0cWFkS3Vyd0J5cC9WUzU1cnBqb2xhQkVGNWlSMGZQcHZzQ1hoTjhKTVUxWHIvYlZ1Y2xDNG9LMC9CSmpNClhwc2Z2ZWwzc0liQWcvTGdLMHYzekJjNm1ncUVJUENyVUpIcHR1VEozZ3RvSVVIR2JWVW00SVN5NFJ0VFlYc2cKSm1UTnc4M0NTd0VRRFV0OVhFUkY0clIxU3RoKzVlVjBWTXBIanB4U2RGa2NwVmo2MzgrQ3UzK0RndWdPYlhqWAorRjBVU2djZ1BscnVwbXkzSHVMb0pxaEk0Z05MYzdsTFVRSURBUUFCb3lNd0lUQU9CZ05WSFE4QkFmOEVCQU1DCkFRWXdEd1lEVlIwVEFRSC9CQVV3QXdFQi96QU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFSN2lZaW9BdmNnS0sKNlNpa1VJZTBlejVZV09vUmxBTitoSlQzV3ozRXp5UHNTc0Y5UnZzckRJVi9xYUkwbWdGMDNCY1B6YWhWSkpHVgpUVEhzVmVGUkVjZnVNdkI5Y3FrU0FTNWprOEZGcUxKUnIvRnZBMEVvYmtRZGZ6eEpzN2hyUmVWUm9tNkZ0bzFJCk9HRXNsTGJlN3c5Q1NFRFdZMkZnRXlEU2hIV05VY1g3TGUzY1ZLMXJDcm10eXl4NzMwbHlCN1lWUjJ2WDcvUGcKNmwrM2t1OHAyMndqcm5KcE9tZ0w0Z2xaWlo2UFZxTlpES3lCWDY3WDlLTGUyays0MWgyTVMzZDdTaWc0bXB0aQo5MUJobnJjQzlhZXlTWUpTNExraWFjV3dDcGFLcGUrcEMwZXpxM2dGT04xS05EQlYvOWRtNzhNMzBNc3RuaU9LCjhYTlFObmhsSnc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+        server: https://api.cloud-platform-live-0.k8s.integration.dsd.io
+      name: cloud-platform-live-0.k8s.integration.dsd.io
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: kuberos
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
Question that's occurred to me before - does it make sense to declare a specific ingress class when we've only got one ingress controller? If we change ingress controller that means we'll need to update any ingress rules that have declared a class, so is it better to not define anything and use the default controller?

Reply:
It very much depends on how we want to handle a transition in the future but I'm not sure there's a clear answer. I would rather not leave it unset since the default behaviour for ingress controller is not necessarily catch-all. At least by having a value, we can better control the transition by adjusting selector flags on the controller. On the other hand, I feel `nginx` is too specific.

Reply:
Fair enough - it's not an issue right now, so we can address it later if we switch ingress controller 👍